support/Pipelines/Bbh/Ringdown.py

```diff
@@ -431,20 +455,28 @@ def start_ringdown(
         " to 0.0"
     ),
 )
+@click.option(
+    "--lev",
+    type=int,
+    help=(
+        "Resolution levels defined in terms of h and p refinement. Can be"
+        " specified multiple times. For integer N, LevN corresponds to"
```

Comment:
Delete "can be specified multiple times" since this isn't implemented yet (could you delete this sentence also from `Inspiral.py`?).